qt/python/mantidqt/widgets/codeeditor/multifileinterpreter.py

```diff
@@ -39,6 +39,7 @@ def __init__(self, default_content=None, parent=None):
 
         # attributes
         self.default_content = default_content
+        self.default_content_filepath = osp.expanduser("~/.mantid/default_editor_content.py")
```

Comment:
Shouldn't this use `mantid.kernel.ConfigService.getUserPropertiesDir()` so it is os independent?